front/src/utils/strings.ts

```diff
@@ -66,3 +66,11 @@ export function convertInputStringToNumber(str: string) {
 export function formatUicToCi(uic: number) {
   return uic.toString().replace(/^87/, '');
 }
+
+export const TEXT_AREA_MAX_LENGTH = 4096;
+export const SMALL_TEXT_AREA_MAX_LENGTH = 1024;
+export const TEXT_INPUT_MAX_LENGTH = 255;
+export const SMALL_INPUT_MAX_LENGTH = 128;
+
+export const stringIsInvalid = (maxLengthAuthorized: number, field?: string | null) =>
+  field ? field.length > maxLengthAuthorized : false;
```

Comment:
The string is invalid if the field is defined (if it is undefined, then it is valid) AND if its length is too long
```suggestion
  field && field.length > maxLengthAuthorized;
```

Reply:
Done

Reply:
Done,but I changed the function a bit like this 
`export const stringIsInvalid = (maxLengthAuthorized: number, field?: string | null): boolean =>
  !!field && field.length > maxLengthAuthorized;` 
 to return a boolean if field is falsy and not get a typescript error

Reply:
Nice !